2019-07-08 17:20:16,593 - root - INFO - Amazon,无url信息或传入参数格式不是列表
2019-07-08 17:20:16,594 - root - INFO - BestBuy,无url信息或传入参数格式不是列表
2019-07-08 17:20:16,594 - root - INFO - JD,无url信息或传入参数格式不是列表
2019-07-08 17:20:16,594 - root - INFO - Tmall,无url信息或传入参数格式不是列表
2019-07-08 17:20:16,614 - root - ERROR - BestBuy(5368400)报错信息：url格式不符合要求
Traceback (most recent call last):
  File "C:\Users\hhh\Desktop\Demo\reviews\bestbuy\bestbuy_review.py", line 163, in run
    self.get_url(url)
  File "C:\Users\hhh\Desktop\Demo\reviews\bestbuy\bestbuy_review.py", line 36, in get_url
    self.max_date = max_date(self.sku_id)
  File "C:\Users\hhh\Desktop\Demo\reviews\utils.py", line 126, in max_date
    max_date = c.execute(sql).fetchone()[0]
cx_Oracle.OperationalError: ORA-01013: 用户请求取消当前的操作
2019-07-08 17:20:16,616 - root - ERROR - BestBuy(5892993)报错信息：url格式不符合要求
Traceback (most recent call last):
  File "C:\Users\hhh\Desktop\Demo\reviews\bestbuy\bestbuy_review.py", line 163, in run
    self.get_url(url)
  File "C:\Users\hhh\Desktop\Demo\reviews\bestbuy\bestbuy_review.py", line 36, in get_url
    self.max_date = max_date(self.sku_id)
  File "C:\Users\hhh\Desktop\Demo\reviews\utils.py", line 126, in max_date
    max_date = c.execute(sql).fetchone()[0]
cx_Oracle.DatabaseError: ORA-24909: 正在执行调用。当前的操作已取消
2019-07-08 17:20:16,617 - root - ERROR - BestBuy(6349969)报错信息：url格式不符合要求
Traceback (most recent call last):
  File "C:\Users\hhh\Desktop\Demo\reviews\bestbuy\bestbuy_review.py", line 163, in run
    self.get_url(url)
  File "C:\Users\hhh\Desktop\Demo\reviews\bestbuy\bestbuy_review.py", line 36, in get_url
    self.max_date = max_date(self.sku_id)
  File "C:\Users\hhh\Desktop\Demo\reviews\utils.py", line 126, in max_date
    max_date = c.execute(sql).fetchone()[0]
cx_Oracle.OperationalError: ORA-03114: 未连接到 ORACLE
2019-07-08 17:20:16,618 - root - INFO - BestBuy_end,耗时0.021970510482788086秒
